app.rb

```diff
@@ -71,7 +90,11 @@ def current_user
   else
     halt 400, "Unknown action: #{payload['action']}"
   end
-  applications = Application.exclude(webhook_url: '').where(pull_request_action => true)
+  applications = Application
+    .where(:legislature => legislatures_in_pr(payload))
+    .or(:legislature => nil)
```

Comment:
Hmm, this doesn't seem to work correctly for newly created webhooks that don't choose a legislature. At the moment they're getting saved to the database with the `legislature` set to `""` rather than `nil`.